widgets/DynamicPartitionedAvroFileSet-batchsink.json

```diff
@@ -0,0 +1,62 @@
+{
+  "metadata": {
+    "spec-version": "1.3"
+  },
+  "configuration-groups": [
+    {
+      "label": "Dynamic Partitioned Fileset - Avro",
+      "properties": [
+        {
+          "widget-type": "dataset-selector",
+          "label": "Dataset Name",
+          "name": "name"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Dataset Base Path",
+          "name": "basePath"
```

Comment:
would it help to have placeholder text like the one used here? https://github.com/hydrator/decompress-action/blob/release/1.0/widgets/Decompress-action.json#L14